README.rst

```diff
@@ -231,14 +231,18 @@ Choose from the list of available rules:
 
 * **binary_operator_spaces** [@Symfony]
 
-  Binary operators should be surrounded by at least one space.
+  Binary operators should be surrounded by space as configured.
 
   Configuration options:
 
-  - ``align_double_arrow`` (``false``, ``null``, ``true``): whether to apply, remove or
-    ignore double arrows alignment; defaults to ``false``
-  - ``align_equals`` (``false``, ``null``, ``true``): whether to apply, remove or ignore
-    equals alignment; defaults to ``false``
+  - ``align_double_arrow`` (``false``, ``null``, ``true``): (deprecated) Whether to apply,
+    remove or ignore double arrows alignment; defaults to ``false``
+  - ``align_equals`` (``false``, ``null``, ``true``): (deprecated) Whether to apply, remove
+    or ignore equals alignment; defaults to ``false``
+  - ``default`` (``'align'``, ``'align_single_space'``, ``'align_single_space_minimal'``,
+    ``'single_space'``, ``null``): default fix strategy; defaults to ``'single_space'``
+  - ``operators`` (``array``): list of binary ``operator`` ``fix strategy`` pairs;
+    defaults to ``[]``
```

Comment:
i got confused, example would be:
```php
[
    ['+', 'align'], // a pair inside list !
]
```

or

```php
[
    '+' => 'align',
]
(a dictionary)
```


?

Reply:
not sure what you want me to put in as copy, it works as requested a few comments back.

Reply:
exactly how you updated it :+1: @SpacePossum